### DISCUSSION TITLE
Using an alternate URL in the client generation process

### DISCUSSION THREAD

**Discussion by caiocinel** [prisma/prisma#22626] [Q&A] [answered]
### Question

Recently, a service provider that I use started charging for external connections to the database, so now we need to use an internal IP to connect to the database, but this IP is not accessible at the time of compiling the project, which causes that Prisma cannot connect to the database and fails to generate.

Is there any way to replace the database URL during the client generation process?

**Comment by nurul3101** [comment]
Hey @caiocinel 👋 

It seems that this GitHub Discussion slipped through the cracks. Do you still need help here?

**Reply by caiocinel** [reply, marked as answer]
I managed to solve it, I ended up getting around the problem by setting the production URL in datasources when the client was initialized, and leaving in .env what would be used for compilation/generation of the client.

**Reply by nurul3101** [reply]
Glad to hear that! 🙏
